sources/packages/web/src/views/aest/AppAEST.vue

```diff
@@ -57,12 +72,14 @@ import BCLogo from "@/components/generic/BCLogo.vue";
 import IdleTimeChecker from "@/components/common/IdleTimeChecker.vue";
 import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
 import CheckPermissionRole from "@/components/generic/CheckPermissionRole.vue";
+import { ref } from "vue";
 
 export default {
   components: { BCLogo, IdleTimeChecker, CheckPermissionRole },
   setup() {
     const { executeLogout } = useAuth();
     const { isAuthenticated } = useAuth();
+    const toggleNav = ref("toggle-nav");
```

Comment:
Please check the other examples in the solution, apparently, it can be declared as `const toggleNav = ref();`.